speaker/main.go

```diff
@@ -354,6 +355,14 @@ func (c *controller) SetConfig(l log.Logger, cfg *config.Config) k8s.SyncState {
 		return k8s.SyncStateErrorNoRetry
 	}
 
+	if c.bgpType == bgpNative {
```

Comment:
bgpNative ?!? u meant bgpFrr ?

Reply:
Nope. The check here is, if native check if something frr only is applied